app/editor/src/features/admin/report-templates/constants/defaultReportTemplate.ts

```diff
@@ -9,8 +9,5 @@ export const defaultReportTemplate: IReportTemplateModel = {
   isEnabled: true,
   sortOrder: 0,
   chartTemplates: [],
-  settings: {
```

Comment:
This is no longer required.  A chart simply displays the content sent to it.  It's up to the report request to include the correct content for each section.